tasks/utils/git.py

```diff
@@ -55,9 +55,12 @@ def git_commit(ctx, target, message):
     ctx.run(cmd)
 
 
-def git_tag(ctx, tag_name):
+def git_tag(ctx, tag_name, push=False):
     """
     Tag the repo using an annotated tag.
     """
     cmd = 'git tag -a {} -m"{}"'.format(tag_name, tag_name)
     ctx.run(cmd)
+    if push:
+        cmd = 'git push --tags'
```

Comment:
I may be missing a piece, but it looks like we are always setting `push` to true? Would we want to allow a "dry run" of this operation before pushing?

Reply:
This is meant to be an utility function so I think "dry run" doesn't apply here, it'd be responsibility of the task using the function to determine the push behaviour, something you can do with `inv release-tag --no-push`